src/libraries/Microsoft.PowerFx.Core/Functions/Delegation/DelegationStrategies/DelegationValidationStrategy.cs

```diff
@@ -63,14 +64,14 @@ protected void SuggestDelegationHint(TexlNode node, TexlBinding binding, ErrorRe
         {
             Contracts.AssertValue(node);
             Contracts.AssertValue(binding);
-            Contracts.Assert(suggestionKey == null || suggestionKey?.Key != string.Empty);
```

Comment:
> string.Empty [](http://example.com/codeflow?start=75&length=13)

Change change from Empty to null?

Reply:
I don't understand your question and this code (looks completely wrong to me)

I've added two exclusions that this fix was supposed to mitigate
[https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1820](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1820)
[https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1508](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1508)